keyboards/akko/5075/keymaps/default/keymap.c

```diff
@@ -0,0 +1,121 @@
+		KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS,KC_EQL,   KC_BSPC,          KC_HOME,
+		KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC,KC_RBRC,  KC_BSLS,          KC_PGUP,
+		KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,KC_NUHS,  KC_ENT,           KC_PGDN,
+		KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_NUBS,         KC_RSFT, KC_UP,   KC_END,
+		KC_LCTL, KC_LALT, KC_LGUI,                   KC_SPC,                             KC_RGUI, MO(MAC_FN),KC_RCTL,        KC_LEFT, KC_DOWN, KC_RGHT),
+
+    [MAC_W] = LAYOUT( /* Base */
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+		_______, _______, KC_UP,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+		_______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, KC_W,    _______,
+		_______, _______, _______,                   _______,                            _______, MO(MAC_FN), _______,       KC_A,    KC_S,    KC_D),
+
+    [MAC_FN] = LAYOUT( /* FN */
+		_______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______,             RGB_MOD,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, RGB_SPD, RGB_SPI, _______,           _______,
+		_______, _______,TG(MAC_W),_______, _______, _______, _______, _______, KC_INS,  _______, KC_PSCR,_______, _______,RGB_MOD,           _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, RGB_TOG, _______, _______, RGB_MOD, RGB_HUI,           _______,
+		_______, _______, _______, KC_CALC, _______, _______, _______, KC_MUTE, KC_VOLD, KC_VOLU, _______, _______,          _______,  RGB_VAI, _______,
+		_______, _______, _______,                   _______,                            _______, _______, _______,          RGB_SAD,  RGB_VAD, RGB_SAI),
+};
+#if defined(ENCODER_MAP_ENABLE)
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+    [WIN_B] = { ENCODER_CCW_CW(KC_VOLU, KC_VOLD) },
+    [WIN_W] = { ENCODER_CCW_CW(KC_VOLU, KC_VOLD) },
+    [WIN_FN] = { ENCODER_CCW_CW(ENC_VAI, ENC_VAD) },
+    [MAC_B] = { ENCODER_CCW_CW(KC_VOLU, KC_VOLD) },
+    [MAC_W] = { ENCODER_CCW_CW(KC_VOLU, KC_VOLD) },
+    [MAC_FN] = { ENCODER_CCW_CW(ENC_VAI, ENC_VAD) }
+};
+#endif
```

Comment:
If this is going to be used, then there has to be a `rules.mk` file inside the `default` keymap folder with `ENCODER_MAP_ENABLE = yes` set.